openiddict/nuget.config

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <packageSources>
+        <add key="nuget" value="https://api.nuget.org/v3/index.json" />
+        <add key="AudaciaPublic" value="https://pkgs.dev.azure.com/audacia/Audacia.Public/_packaging/AudaciaPublic/nuget/v3/index.json" />
+    </packageSources>
+
+    <packageSourceMapping>
```

Comment:
This is required when using Central Package Management, the build fails without it - https://learn.microsoft.com/en-us/nuget/consume-packages/central-package-management#warning-when-using-multiple-package-sources

However, there is an issue with this when using the DevOps `DotNetCoreCLI@2` `restore` task - https://github.com/microsoft/azure-pipelines-tasks/issues/15542#issuecomment-1129700406

Maybe we could pass a flag into the the Audacia.Build templates to change how the restore works? Would need to pass through quite a few templates though

Reply:
If this is too complex, maybe we revert the nice-to-have central package management and go back to managing the versions in each csproj - this will mean copying the conditions into each csproj that needs package versions changing.

It does look like EFCore is the only versions changing between .NET 6 and .NET 8 so maybe this will be the easiest fix

Reply:
Have removed this now no longer doing central package management.